### ISSUE TITLE
Node shapes not always drawn in pgf output

### ISSUE THREAD

**Issue by avengerpenguin** [kjellmf/dot2tex#44]
So, I have this graph:

``` dot
      digraph g {
        rankdir=LR;

        node [shape=circle,margin="0,0"];

        URI [label="URI"];
        RDF1 [label="RDF_1"];
        RDF2 [label="RDF_2"];
        RDFp [label="RDF'"];

        URI -> RDFa [label="fetch"];
        RDFa -> RDF [label="parse"];
        RDF -> RDF1 [label="DBPedia"];
        RDF -> RDF2 [label="DBPedia"];
        RDF -> RDFp [label="\cup"];
        RDF1 -> RDFp [label="\cup"];
        RDF2 -> RDFp [label="\cup"];
        RDFp -> Features [label="generate"];
      }
```

which when run with `dot2tex -f pgf` does not output any `\draw ellipse` commands and I get no circles around my nodes when I embed via the dot2texi package.

However, if I just add a random node at the start:

``` dot
      digraph g {
        rankdir=LR;

        node [shape=circle,margin="0,0"];

        banana;

        URI [label="URI"];
        RDF1 [label="RDF_1"];
        RDF2 [label="RDF_2"];
        RDFp [label="RDF'"];

        URI -> RDFa [label="fetch"];
        RDFa -> RDF [label="parse"];
        RDF -> RDF1 [label="DBPedia"];
        RDF -> RDF2 [label="DBPedia"];
        RDF -> RDFp [label="\cup"];
        RDF1 -> RDFp [label="\cup"];
        RDF2 -> RDFp [label="\cup"];
        RDFp -> Features [label="generate"];
      }
```

then I do get circles in my document via the dot2texi environment and I do see things like `\draw ellipse` for each node in the pgf output. Can anyone explain that?

I've got some simpler versions of the diagram earlier in the document that work fine, but there's some strange combination of things in my first file above where all the node shapes just disappear. I've actually seen this a few times with very different diagrams, but this is the first time I've tried `dot2tex` directly to see the output changes before it even gets to be embedded in my overall LaTeX document.

**Comment by expbbc** [comment]
This was fixed in 2014 here: https://github.com/kjellmf/dot2tex/commit/914f8051e7a64ef118d2cb7a355b0011ec15f970

Kjellmf has never bothered to push an updated version to pip.

**Comment by avengerpenguin** [comment]
Has the project gone stale then?

**Comment by kjellmf** [comment]
Fixed in #60
